compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/scene/SwingComposeSceneLayer.desktop.kt

```diff
@@ -72,7 +79,7 @@ internal class SwingComposeSceneLayer(
         it.isOpaque = false
         it.background = Color.Transparent.toAwtColor()
         it.size = Dimension(windowContainer.width, windowContainer.height)
-        it.addMouseListener(this)
+        it.addMouseListener(backgroundMouseListener)
 
```

Comment:
Another bug, not for this PR. We have blinking at the start:
<img width="886" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/f7282a43-add4-43b4-952f-294d2a35b576">
<img width="886" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/858acade-209c-4f62-8073-c1fef18b16eb">
<img width="888" alt="image" src="https://github.com/JetBrains/compose-multiplatform-core/assets/5963351/758cef86-2d55-4f5b-ad04-0554d82c7684">

Reply:
https://github.com/JetBrains/compose-multiplatform/issues/4475